lib/cog_api/http/client.ex

```diff
@@ -124,6 +124,14 @@ defmodule CogApi.HTTP.Client do
     RelayGroups.remove_relay(relay_group_id, relay_id, endpoint)
   end
 
+  def relay_group_assign_bundle(relay_group_id, bundle_id, %Endpoint{}=endpoint) do
```

Comment:
What do you think about trying to choose a consistent in the naming and using `relay_group_add_bundle`. They mean essentially the same thing and then you have less guess work about how to add a bundle to a relay group.